.github/workflows/ci.yml

```
@@ -286,3 +286,59 @@ jobs:
         with:
             name: glpi-logs
             path: tests/files/_log
+
+  playwright:
+    # Do not run scheduled tests on tier repositories
+    if: github.repository == 'glpi-project/glpi' || github.event_name != 'schedule'
+    name: "Playwright tests using PHP 8.5 and MariaDB 11.8"
```

PHP/DB versions are not an important version here.

```suggestion
    name: "Playwright tests"
```